### ISSUE TITLE
Support for multiple request body types

### ISSUE THREAD

**Issue by jwag956** [mrin9/RapiDoc#26]
First and foremost - this is an awesome tool - and produces better output than anything else I have found.. and just works. So thank you.

My API supports both application/json and application/x-www-form-urlencoded for some APIs. So I modeled it as:
```
    post:
      summary: Login to application
      requestBody:
        required: true
        content:
          application/json:
            schema:
              $ref: "#/components/schemas/Login"
          application/x-www-form-urlencoded:
            schema:             
              $ref: "#/components/schemas/Login"
```

This always renders as 'FORM DATA' and doesn't show the json request model.
Responses seems to work and render fine with multiple response types.

**Comment by mrin9** [comment]
can you put a screenshot of whats wrong and what you are expecting.
When I test with petstore example, which accepts multiple content-type in request-body (application/json and application/xml)

On the right top corner of the request body parameter input box it will show both the options

**Comment by jwag956** [comment]
I was trying to use openapi 3.0.0 - possibly that is my problem - I am finding pretty minimal support for that across the board. I tried petstore and see the POST /pet seems to work as I would expect.. so maybe this is 3.0.0 related?

Thanks for your help.

But - here is the relevant portion of yaml and attached screen shot.

```
  /register:
    get:
      summary: Retrieve register form
      responses:
        200:
          description: Register form
          content:
            text/html:
              schema:
                example: render_template(config_value('REGISTER_USER_TEMPLATE')
    post:
      summary: Register with application
      description: Supports both json and form request types
      parameters:
        - name: next
          in: query
          description: >
              URL to redirect to on successful registration. Ignored for json request.
          schema:
            type: string

      requestBody:
        required: true
        content:
          application/json:
            schema:
              $ref: "#/components/schemas/Register"
          application/x-www-form-urlencoded:
            schema:             
              $ref: "#/components/schemas/RegisterForm"
      responses:
        200:
          description: Register response
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/DefaultJsonResponse"
            text/html:
              schema:
                description: Unsuccessful registration
                example: render_template(config_value('REGISTER_USER_TEMPLATE') with error values
        302:
          description: >
            Successful registration with form data body.
            redirect('next') or redirect(config_value('POST_REGISTER_VIEW'))
          headers:
            Location:
              description: redirect
              schema:
                type: string
                example: redirect(config_value('POST_REGISTER_VIEW'))
        400:
          description: Errors while validating registration form
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/DefaultJsonErrorResponse"  
```

![Screen Shot 2019-05-15 at 3 37 37 PM](https://user-images.githubusercontent.com/1002831/57814280-0a7ef380-7728-11e9-9a62-326bc9a94ede.png)

**Comment by mrin9** [comment]
I c what you mean. This shouldn't be your problem of using openapi 3. 
In RapiDoc, We incorrectly assumed that the request body can either be `form` or `json / xml` but not both

We need to think of some UI changes also to accommodate this. Coz of the `TRY` feature where the body can be one of them when you submit the request.

Thanks for reporting this, We will try to come up with some solution but cannot promise an immediate fix.

**Comment by jwag956** [comment]
Thanks for the quick analysis...
FWIW - I am not using the TRY feature and I can believe that would make that much more complex.

**Comment by abeckers** [comment]
Hi, I need this feature too. Any timeline for it?

**Comment by adamdecaf** [comment]
I'm having a related issue where the requestBody examples don't change when toggling between content samples. The petstore shows an `<input>` but nothing changes after selecting the other type.

<img width="1068" alt="Screen Shot 2020-03-19 at 2 05 35 PM" src="https://user-images.githubusercontent.com/120951/77115006-c49b5780-69ea-11ea-9a9e-6e49524ecf3e.png">

On my OpenAPI spec I see `<input type="radio" />`, but nothing changes after selecting between options. See: https://api.moov.io/apps/ach/#post-/files/create

**Comment by mrin9** [comment]
@jwag956 
Support for multiple request body is added in release `7.5.0`.
See it in action here https://mrin9.github.io/RapiDoc/examples/request-body-multiple.html

@adamdecaf 
Support for multiple examples in request-body and elsewhere is also addressed in release `7.5.0`
check it out here https://mrin9.github.io/RapiDoc/examples/examples-multiple.html

**Comment by adamdecaf** [comment]
Thank you!
